### PR TITLE
[search] Followup fixes to TestSearchRequest.

### PR DESCRIPTION
**Pull request by mpimenov**
@ygorshenin PTAL

### CODE REVIEW

File: search/search_tests_support/test_search_request.cpp [hunk 1]
```diff
@@ -71,18 +61,17 @@ vector<search::Result> const & TestSearchRequest::Results() const
   return m_results;
 }
 
-void TestSearchRequest::SetUpCallbacks()
+void TestSearchRequest::Start() { m_engine.Search(m_params, m_viewport); }
+void TestSearchRequest::Wait()
```
Comment (ygorshenin) [inline search/search_tests_support/test_search_request.cpp]: Blank line, please.

Comment (mpimenov) [inline search/search_tests_support/test_search_request.cpp]: done

**Comment by mpimenov** [comment]
@ygorshenin PTAL

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**